plugins/topology/README.md

```diff
@@ -0,0 +1,107 @@
+# Topology plugin for Backstage
+
+This plugin will help with visualizing the workloads like Deployment, Replicasets, Pods powering any serivce on kubernets cluster.
+
+## Prerequisites
+
+1. Install and configure kubernetes plugin by following these guides:
+   https://backstage.io/docs/features/kubernetes/installation
+   https://backstage.io/docs/features/kubernetes/configuration
+2. Kubernetes plugin is properly configured and able to connect to the cluster via a `ServiceAccount`
+3. Following `ClusterRole` must be granted to ServiceAccount accessing the cluster: (k8s plugin by default tries to fetch the following resources and if access is not provided an error is shown)
```

Comment:
I think we should mention this that this role is already granted if you have the Backstage Kubernetes Plugin properly configured (which is a prerequisite), and link to https://backstage.io/docs/features/kubernetes/configuration#role-based-access-control

The current formulation gives impression that there is something different in the role itself when compared to the one required by kubernetes plugin. While that may become the case in future, I think we should rather formulate in a way that basically says: "Please grant the permissions that Backstage Kubernetes plugin requires + these additional ones" (once we have any additional ones).